docs/governance/security-template.md

```diff
@@ -0,0 +1,142 @@
+[//]: # (SPDX-License-Identifier: CC-BY-4.0)
+# ==PROJECT== Security Policy
+
+## Instructions
+
+The following is the best practices template security vulnerability disclosure
+policy for OpenWallet Foundation's projects. Please copy the text below, place it into the `SECURITY.md` file for the primary repository of your project,
+and adjust it as necessary for your project. Notably:
+
+* Remove this "Instructions" section.
+* Replace ==PROJECT== in the title of this page with the name of your project.
+* Populate the [Security Team section](#security-team) with maintainers who have agreed to be on the security team.
+* See the "**Alternative**" notes in the [OpenWallet Foundation's security vulnerability disclosure policy] document for how the "best practices" in this document can be changed to meet the needs of your project while still adhering to the OpenWallet Foundation's policies.
+
+Once your project's security vulnerability disclosure policy document is
+published, place a copy of it into each of your project's repositories, or
```

Comment:
The `or` in this sentence "...place a copy of it into each of your project's repositories, or
create a `SECURITY.md`" implies that there are other ways to "place it in the repo".

I would suggest for simplicity to give only one option ant propose the standard way of using the `SECURITY.md` file and rephrase it to

> ...place a copy of it into each of your project's repositories by creating a `SECURITY.md`

Reply:
Thoughts on rephrasing this as:

> published, place a copy of it into each of your project's repositories (or link to the main repository's SECURITY.md).

The thing that I want to avoid is having a project have the same `SECURITY.md` file in multiple repositories and then changing only one of them (when they should all be changed).

Reply:
Fixed in [commit c780210](https://github.com/openwallet-foundation/tac/pull/92/commits/c78021093555e24c3e9ff13d75da3183296804d8)